stylesheet.css

```diff
@@ -1,146 +1,155 @@
-body{
-    background-color: #161316;
-}
-
-.mainBannerContainer{
-    position: auto;
-    display: block;
-    width: 1300px;
-    height: 780px;
-    margin-top: 20px;
-    margin-left:auto;
-    margin-right:auto;
-    margin-bottom: 0;
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
 }
 
-.mainBodyContainer{
-    position: auto;
-    display: block;
-    width: 1300px;
-    height: auto;
-    margin-top: 0px;
-    margin-left:auto;
-    margin-right:auto;
-    margin-bottom: 0;
-    /* border-style: solid; 
-    border-color: aliceblue; */
+:root {
+  --grey: #c6c5c5;
+  --white: #ffffff;
+  --header: #2da4af;
+}
+
+body {
+  margin: 0 auto;
+  background-color: #161316;
+  max-width: 1300px;
+}
+
+body * {
+  /* border: 1px solid red; */
+}
+
+.mainBannerContainer {
+  margin-top: 20px;
+  padding: 8px;
+  position: relative;
 }
 
 .bannerImage {
-    position: relative;
-    display: block;
-    width: 1300px;
-    border-radius: 48px;
+  width: 100%;
+  border-radius: 48px;
 }
 
-/* Aleksandra Georgievska Line Header text */
-.AGHeaderPosition {
-    position: relative;
-    display: block;
-    width: 700px;
-    top: -490px;
-    left: 30px;
-    opacity: 85%;
+.bannerCopy {
+  position: absolute;
+  top: 28%;
+  color: var(--grey);
+  font-family: monospace;
+  margin-left: 20px;
 }
 
-/* Former Pro Musician | Aspiring MLE Line Header text */
-.subHeaderPosition {
-    position: relative;
-    display: block;
-    width: 900px;
-    top: -470px;
-    left: 35px;
-    opacity: 85%;
+.title {
+  font-size: 48px;
 }
 
+.subtitle {
+  font-size: 24px;
+  margin-top: 12px;
+}
+
+.socialLinks {
+  display: inline-block;
+  position: absolute;
+  top: 8%;
+  right: 4%;
+  width: 64px;
+}
+
+.socialLink {
+  width: 48px;
+}
+@media (min-width: 900px) {
+  .socialLink {
+    width: 64px;
+  }
+}
 
 .gitHubPosition {
-    position: relative;
-    display: block;
-    width: 65px;
-    top: -790px;
-    left: 1200px;
-    opacity: 85%;
+  /* make these icons instead of pngs. Icons are easy to style and are a smaller filesize. */
+  opacity: 85%;
```

Comment:
If this was an icon you could adjust your `fill` property to make the colors match.